dm/worker/subtask.go

```diff
@@ -364,56 +382,8 @@ func (st *SubTask) Close() {
 
 	st.cancel()
 	st.closeUnits() // close all un-closed units
-	st.setStageIfNot(pb.Stage_Finished, pb.Stage_Stopped)
+	st.setStageIfNot(pb.Stage_Finished, pb.Stage_Paused)
 	st.wg.Wait()
-
-	close(st.DDLInfo)
```

Comment:
do it in Init

Reply:
in `Init()`, we only create a new one, by not close the old one?

Reply:
`gc` would take over it, and I really don't know whether it would affect `SendBackDDLInfo` and what's useful of `SendBackDDLInfo`

Reply:
previously, if the DM-master resolve the DDL lock failed, the DDL info will be sent back to DM-worker, so after DM-master restarted it can fetch DDL info from DM-worker again. but now `SendBackDDLInfo` is useless, because `SaveDDLInfo` introduced...

I think the better solution is to store the DDL info in levelDB, and we can do it later.